### ISSUE TITLE
Question, where to view the help doc of the custom shared lib

### ISSUE THREAD

**Issue by missedone** [mschuchard/jenkins-devops-libs#1]
you created some nice shared libs like interacting with helm, i see there's a help doc https://github.com/mschuchard/jenkins-devops-libs/blob/master/vars/helm.txt

where can I see this rendered help doc on Jenkins?

> A variable defined in a shared library will only show up in Global Variables Reference (under Pipeline Syntax) after Jenkins loads and uses that library as part of a successful Pipeline run.

i didn't see the lib even build success.
Thanks

**Comment by mschuchard** [comment]
Well that is good news, because I had been unable to get the docs to appear and had put troubleshooting that on my task list. Glad you figured this one out.

**Comment by missedone** [comment]
to be more specific, the help doc is showing up on the job pipeline-syntax page, Ex,
`https://<jenkins_host>/job/<your_job>/pipeline-syntax/globals`

you must have a success build with the libs, then the help doc of the libs will show up.

**Comment by mschuchard** [comment]
Yeah I was unable to get the doc to appear there and had assumed I had misnamed the file or something else simple. Apparently restarting the Jenkins instance is the magic trick there. Good find.
